metadata.json

```diff
@@ -10,19 +10,19 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 9.0.0 < 10.0.0"
+      "version_requirement": ">= 5.2.0 < 10.0.0"
     },
     {
       "name": "puppetlabs/apt",
       "version_requirement": ">= 2.0.0 < 10.0.0"
     },
     {
       "name": "puppet/systemd",
-      "version_requirement": ">= 4.0.1 < 6.0.0"
+      "version_requirement": ">= 4.0.1 < 5.0.0"
     },
     {
       "name": "puppetlabs/concat",
-      "version_requirement": ">= 4.1.0 < 10.0.0"
+      "version_requirement": ">= 4.1.0 < 9.0.0"
```

Comment:
I don't think you need to revert these 2.

Reply:
I will revert those two. That allows us to skip this PR and the original one in the changelog.

Reply:
ah no! those were added in another PR so we don't need to revert it. #1448